src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
You can remove this file and similar files being used for testing as it is out of the scope of this project